Libraries/NavigationExperimental/NavigationPropTypes.js

```diff
@@ -59,14 +59,14 @@ const scene = PropTypes.shape({
 });
 
 /* NavigationSceneRendererProps */
-const SceneRenderer = {
+const SceneRenderer = PropTypes.shape({
```

Comment:
Flow now fails due to this change. This object is being spread in `NavigationCard`.

The reason I changed it because otherwise the `NavigationPropTypes` exported from this cannot be used directly like `PropTypes.shape(NavigationPropTypes)` or even by spreading, since it contains a mix of PropTypes and plain objects. What do you suggest I do here?